Tests/id_set.json

```diff
@@ -194,6 +200,9 @@
                     "abuseipdb-get-blacklist", 
                     "createNewIndicator"
                 ], 
+                "script_executions": [
+                    "createNewIndicator"
```

Comment:
why there are `createNewIndicator` two times?

Reply:
Because it matches the pattern of a script, should be much of a difference if we remove it, i think it is insignificant